zha/application/platforms/select.py

```diff
@@ -637,6 +637,25 @@ class InovelliNonNeutralOutputEntity(ZCLEnumSelectEntity):
     _attr_translation_key: str = "increased_non_neutral_output"
 
 
+class InovelliDimmingMode(types.enum1):
+    """Inovelli dimming mode selection."""
+
+    LeadingEdge = 0x00
+    TrailingEdge = 0x01
```

Comment:
We should start migrating the Inovelli attributes in the quirk to the new Zigpy format, this will keep enums like this out of ZHA:

```python
class LeadingOrTrailingEdge(t.enum8):
    LeadingEdge = 0x00
    TrailingEdge = 0x01


class InovelliVZM31SNCluster(CustomCluster):
    name = "InovelliVZM31SNCluster"

    class AttributeDefs(BaseAttributeDefs):
        """Attribute definitions."""
        ...

        leading_or_trailing_edge = ZCLAttributeDef(
            id=0x001A,
            type=LeadingOrTrailingEdge,
            zcl_type=DataTypeId.bool, # It's really a bool on the device
            is_manufacturer_specific=True,
        )

        ...
```